ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -16,80 +16,90 @@
 
 package io.confluent.ksql.rest.server.computation;
 
-import org.apache.kafka.clients.consumer.KafkaConsumer;
-import org.apache.kafka.clients.producer.KafkaProducer;
-import org.apache.kafka.common.serialization.Deserializer;
-import org.apache.kafka.common.serialization.Serializer;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.common.TopicPartition;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
-import io.confluent.kafka.serializers.KafkaJsonDeserializer;
-import io.confluent.kafka.serializers.KafkaJsonSerializer;
-import io.confluent.ksql.metastore.MetaStoreImpl;
 import io.confluent.ksql.rest.entity.CommandStatus;
-import io.confluent.ksql.rest.server.StatementParser;
-import io.confluent.ksql.rest.server.mock.MockCommandStore;
-import io.confluent.ksql.rest.server.mock.MockKafkaTopicClient;
-import io.confluent.ksql.rest.server.mock.MockKsqkEngine;
 import io.confluent.ksql.rest.server.utils.TestUtils;
+import io.confluent.ksql.util.Pair;
+
+import static org.easymock.EasyMock.*;
 
 public class CommandRunnerTest {
 
+  /**
+   *   Get a command runner instance using mock values
+    */
   private CommandRunner getCommanRunner(StatementExecutor statementExecutor) {
+    List<Pair<CommandId, Command>> commandList = new TestUtils().getAllPriorCommandRecords();
+    List<ConsumerRecord<CommandId, Command>> recordList = new ArrayList<>();
+    for (Pair commandPair: commandList) {
+      recordList.add(new ConsumerRecord<CommandId, Command>("T", 1, 1, (CommandId) commandPair
+          .getLeft(), (Command) commandPair.getRight()));
+    }
+    Map<TopicPartition, List<ConsumerRecord<CommandId, Command>>> recordMap = new HashMap<>();
+    recordMap.put(new TopicPartition("T", 1), recordList);
+    CommandStore commandStore = mock(CommandStore.class);
+    expect(commandStore.getNewCommands()).andReturn(new ConsumerRecords<>(recordMap));
```

Comment:
These expectations are misleading. `CommandStore#getNewCommands` is only called in the context of `fetchAndRunCommands`. `CommandStore#getPriorCommands` is only called in the context of `processPriorCommands`. So the relevant expectations should be set in the relevant test.

Reply:
Moved the expected results to the corresponding test.